internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/VeniceTestListener.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.venice;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+
+
+public class VeniceTestListener extends TestListenerAdapter {
```

Comment:
Note that we already have `com.linkedin.venice.testng.VeniceSuiteListener`... should we put this new listener in the same package?

Also, I see that it is commented out in the build. Is this intended? Shall we keep it? If we do keep it but want it to be disabled by default, then I think we should leave a comment in the build to indicate when it might be useful to uncomment it. What do you think?

Reply:
>Note that we already have com.linkedin.venice.testng.VeniceSuiteListener... should we put this new listener in the same package?

Initially, I created this class in `com.linkedin.venice.testng` but somehow it was getting picked up. Let me give it another shot. 

> Also, I see that it is commented out in the build. Is this intended? Shall we keep it? If we do keep it but want it to be disabled by default, then I think we should leave a comment in the build to indicate when it might be useful to uncomment it. What do you think?

It would be good to have `VeniceTestListener` as it adds nice markers in logs that indicate the start and end of a test.

When `VeniceTestListener` is enabled some of our tests start failing and that's why I disabled it to see if `VeniceTestListener` was indeed the reason. 


I'm still doing some experiments. If `VeniceTestListener` doesn't work well I'll get rid of it. 

Thanks.

Reply:
Seem like `VeniceTestListener` is working as expected.